config/nginx/default.conf

```diff
@@ -24,6 +24,13 @@ server {
 		fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
 	}
 
+	location = /robots.txt {
+		return 200 'User-agent: *
+Disallow: /';
+		access_log off;
+		log_not_found off;
+	}
+
 	client_max_body_size 100m;
-	include includes/wpsubdir.conf;
+	include /etc/nginx/sites-enabled/*;
```

Comment:
why we need this? Every app will be a single nginx? Or is this related to front-end proxy?